src/setup/cortx_setup.py

```diff
@@ -52,12 +53,12 @@ def _validate(self):
         if self._args.action not in ['apply']:
             raise CortxProvisionerError(errno.EINVAL, 'Invalid action type')
 
-        if self._args.log_level:
-            if self._args.log_level not in \
-                ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']:
-                    raise CortxProvisionerError(errno.EINVAL, 'Invalid log level')
-            else:
-                Log.logger.setLevel(self._args.log_level)
+        log_level = self._args.log_level
+        if not log_level:
+            log_level = os.getenv('CORTX_PROVISIONER_DEBUG_LEVEL', 'INFO')
```

Comment:
Change INFO to DEFAULT_LOG_PATH constant

Reply:
Done.